[Bug] tracing.service.ts에서 SDK 서브패스 import로 인한 전역 설치 오류

## 문제
`npm i -g crewx` 후 실행 시 모듈 해상도 오류:
```
Error: Cannot find module '../../../sdk/dist/config/pricing'
```

## 원인
`packages/cli/src/services/tracing.service.ts`에서 서브패스 import 사용:
```typescript
import { calculateCost } from '@sowonai/crewx-sdk/config/pricing';
```

## 해결 방법
SDK 메인 index에서 이미 `calculateCost`를 export하고 있으므로, 정석대로 수정:
```typescript
import { calculateCost } from '@sowonai/crewx-sdk';
```

## 작업 범위
1. `tracing.service.ts` import 경로 수정
2. 빌드 확인
3. rc.22 배포 및 전역 설치 테스트

## 참고
- PR #94 머지 후 발견된 런타임 오류
- 이슈 #93 후속 작업

--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "crewx-monorepo",
   "private": true,
-  "version": "0.8.0-rc.20",
+  "version": "0.8.0-rc.21",
   "license": "Apache-2.0",
   "description": "CrewX monorepo workspace",
   "workspaces": [
@@ -26,7 +26,7 @@
     "clean": "rimraf node_modules \"packages/*/node_modules\"",
     "bump:dev": "node ./scripts/dev-bump.mjs",
     "changeset": "changeset",
-    "version": "0.8.0-rc.20",
+    "version": "0.8.0-rc.21",
     "release": "npm run build && npm run test && changeset publish",
     "prerelease": "npm run build && npm run test"
   },

--- a/packages/cli/package.json
+++ b/packages/cli/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@sowonai/crewx-cli",
-  "version": "0.8.0-rc.20",
+  "version": "0.8.0-rc.21",
   "license": "MIT",
   "engines": {
     "node": ">=20.19.0"
@@ -72,7 +72,7 @@
     "@nestjs/platform-express": "^11.0.0",
     "@slack/bolt": "^4.4.0",
     "@slack/web-api": "^7.10.0",
-    "@sowonai/crewx-sdk": "^0.8.0-rc.20",
+    "@sowonai/crewx-sdk": "^0.8.0-rc.21",
     "@sowonai/nestjs-mcp-adapter": "^0.1.3",
     "@types/js-yaml": "^4.0.9",
     "@types/mdast": "^4.0.4",

--- a/packages/cli/package.json
+++ b/packages/cli/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@sowonai/crewx-cli",
-  "version": "0.8.0-rc.21",
+  "version": "0.8.0-rc.22",
   "license": "MIT",
   "engines": {
     "node": ">=20.19.0"
@@ -72,7 +72,7 @@
     "@nestjs/platform-express": "^11.0.0",
     "@slack/bolt": "^4.4.0",
     "@slack/web-api": "^7.10.0",
-    "@sowonai/crewx-sdk": "^0.8.0-rc.21",
+    "@sowonai/crewx-sdk": "^0.8.0-rc.22",
     "@sowonai/nestjs-mcp-adapter": "^0.1.3",
     "@types/js-yaml": "^4.0.9",
     "@types/mdast": "^4.0.4",

--- a/packages/cli/src/services/tracing.service.ts
+++ b/packages/cli/src/services/tracing.service.ts
@@ -2,7 +2,7 @@ import { Injectable, Logger, OnModuleDestroy, OnModuleInit, Optional, Inject } f
 import * as fs from 'fs';
 import * as path from 'path';
 import Database from 'better-sqlite3';
-import { calculateCost } from '@sowonai/crewx-sdk/config/pricing';
+import { calculateCost } from '@sowonai/crewx-sdk';
 
 /**
  * Task status enum

--- a/packages/crewx/package.json
+++ b/packages/crewx/package.json
@@ -1,6 +1,6 @@
 {
   "name": "crewx",
-  "version": "0.8.0-rc.20",
+  "version": "0.8.0-rc.21",
   "license": "MIT",
   "engines": {
     "node": ">=20.19.0"
@@ -22,7 +22,7 @@
     "test:unit": "npm run test"
   },
   "dependencies": {
-    "@sowonai/crewx-cli": "0.8.0-rc.20"
+    "@sowonai/crewx-cli": "^0.8.0-rc.21"
   },
   "publishConfig": {
     "access": "public"

--- a/packages/sdk/package.json
+++ b/packages/sdk/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@sowonai/crewx-sdk",
-  "version": "0.8.0-rc.20",
+  "version": "0.8.0-rc.21",
   "license": "Apache-2.0",
   "engines": {
     "node": ">=20.19.0"

--- a/packages/sdk/package.json
+++ b/packages/sdk/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@sowonai/crewx-sdk",
-  "version": "0.8.0-rc.21",
+  "version": "0.8.0-rc.22",
   "license": "Apache-2.0",
   "engines": {
     "node": ">=20.19.0"

